### DISCUSSION TITLE
Wordfence Security reports vulnerability after last plugin update

### DISCUSSION THREAD

**Discussion by DCO84770** [code4recovery/12-step-meeting-list#1343] [Q&A] [answered]
Hello
We use Wordefence Security and it reports a vulnerability after we just updated the plugin. WP version is 6.4.3. See:
https://www.wordfence.com/threat-intel/vulnerabilities/wordpress-plugins/12-step-meeting-list/12-step-meeting-list-31426-missing-authorization
No functionality issues seen.
Thanks

**Comment by dvcb** [comment, marked as answer]
@DCO84770 We've been in touch with Wordfence about this erroneous report.  They updated their database this morning to correct the problem. We've confirmed that at least one site using Wordfence Pro is no longer reporting a vulnerability. Please try doing another full Wordfence scan and let us know if the problem persists.

**Reply by DCO84770** [reply]
Thanks!
